build/transforms/Dashboard.config.install.xdt

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<dashBoard xmlns:xdt="http://schemas.microsoft.com/XML-Document-Transform">  
+  <section alias="Our.Umbraco.BackofficeThemes"
+            xdt:Locator="Match(alias)"
+            xdt:Transform="InsertIfMissing">
+			<areas>
+				  <area>developer</area>
+			</areas>
+			<tab caption="Backend Theme">
```

Comment:
Yeah, I like this transform. We just need to keep consistent names, so please rename it for now to "Backoffice Themes" to follow the package name. Also.. the good thing to discuss in this area is to where it should be placed, as it might be good to expose it for "normal" editors as well, not only those who has access to the Developer's section.

Reply:
Fixed in latest Commit